### ISSUE TITLE
Exception trying to select Browse Remote hosts

### ISSUE THREAD

**Issue by kamal2222ahmed** [sveale/remote-edit#89]
i selected Packages --> Remote Edit --> "Browse remote host"

Got exception:

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.4
**Thrown From**: [remote-edit](git://github.com/sveale/remote-edit) package, v1.8.4

### Stack Trace

Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is no longer a child of this node. Perhaps it was moved in a 'blur' event handler?

```
At /Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/src/panel-container-element.js:63

Error: Failed to execute 'removeChild' on 'Node': The node to be removed is no longer a child of this node. Perhaps it was moved in a 'blur' event handler?
  at Error (native)
  at atom-panel-container.PanelContainerElement.panelRemoved (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/src/panel-container-element.js:63:19)
  at Emitter.module.exports.Emitter.emit (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at PanelContainer.module.exports.PanelContainer.panelDestroyed (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/src/panel-container.js:93:29)
  at Emitter.module.exports.Emitter.emit (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at Panel.module.exports.Panel.destroy (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/src/panel.js:25:20)
  at HostsView.module.exports.HostsView.destroy (/Users/a3x52zz/.atom/packages/remote-edit/lib/view/hosts-view.coffee:24:14)
  at HostsView.module.exports.HostsView.cancelled (/Users/a3x52zz/.atom/packages/remote-edit/lib/view/hosts-view.coffee:33:8)
  at HostsView.module.exports.HostsView.cancel (/Users/a3x52zz/.atom/packages/remote-edit/lib/view/hosts-view.coffee:28:8)
  at HostsView.module.exports.HostsView.confirmed (/Users/a3x52zz/.atom/packages/remote-edit/lib/view/hosts-view.coffee:85:8)
  at HostsView.module.exports.SelectListView.confirmSelection (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-li.<anonymous> (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:139:19)
  at HTMLOListElement.jQuery.event.dispatch (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLOListElement.elemData.handle (/Users/a3x52zz/Downloads/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -5:35.7.0 core:copy (div.package-readme.native-key-bindings)
     -3:43.1.0 remote-atom:start-server (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:51.6.0 remote-edit:new-host-sftp (atom-text-editor.editor.is-focused)
     -0:41.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:22.7.0 jenkins:list (atom-text-editor.editor.mini.is-focused)
     -0:05.8.0 remote-edit:browse (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
Handlebars, v1.1.2
grunt-runner, v0.10.0
jenkins, v0.3.4
language-gradle, v0.0.3
linter-perl, v0.5.1
minimap, v4.10.0
perltidy, v2.2.0
pigments, v0.8.1
remote-atom, v1.2.6
remote-edit, v1.8.4
script-runner, v1.8.3

# Dev
No dev packages
```

**Comment by sveale** [comment]
Have you tried restarting atom or reinstalling the package?

**Comment by lgrn** [comment]
I just got this as well after updating from 1.8.4 to 1.8.5 (not sure if it's relevant). Connection worked fine before upgrading.

I'm also seeing some very weird behavior when I try to delete an added SFTP server with Ctrl+d -- after deleting it and pressing [ESC] I get the above error again, and when I press Ctrl+Alt+b, nothing has actually been deleted.

**Comment by lgrn** [comment]
This bug appeared for me when moving from 1.8.4 -> 1.8.5. I deleted the package (got some buggy behavior when doing that but it was gone when Atom was restarted) and reinstalled it with apm, and that solved the issue. The servers saved previously were preserved.

**Comment by sveale** [comment]
Should be fixed in commit e6afcf530fc0287508286707c16afd7577501894.
